shared/src/MessageIdDefault.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// The different configurations available for setting a default value of MessageId on an IoT Hub Message.
+    /// </summary>
+    public enum MessageIdDefault
+    {
+        /// <summary>
+        /// MessageId is settable only by the user.
```

Comment:
set, not settable. It isn't that the SDK can't, it just won't.

Reply:
That's true